content/docs/1.6.0/deploy/important-notes/index.md

```diff
@@ -100,6 +100,9 @@ When upgrading through Helm, a component compatibility check is automatically pe
 
 If you installed Longhorn using the manifests, engine upgrades are enforced by the Longhorn Manager. Attempts to upgrade Longhorn Manager may cause unsuccessful pod launches and generate corresponding error logs, although it poses no harm. If you encounter such errors, you must revert to the previous Longhorn version and then upgrade the engines that are using the incompatible engine images before the next upgrade.
 
+> **Warning:**
+> Longhorn enables recovery to the previous version in case of upgrade failure caused by the engine upgrade enforcement. This is because Longhorn manager will fail to block the entire upgrade. However, downgrading is not supported once an upgrade is successful. For more information, refer to [Upgrade Path Enforcement](../../deploy/upgrade/#upgrade-path-enforcement).
```

Comment:
```suggestion
> Whenever engine upgrade enforcement causes upgrade failure, Longhorn allows you to revert to the previous version because Longhorn Manager does not block the entire upgrade. However, Longhorn prohibits downgrading when an upgrade is successful. For more information, see [Upgrade Path Enforcement](../../deploy/upgrade/#upgrade-path-enforcement).
```

Reply:
I used "revert" because that is the term used in Upgrade Path Enforcement. Also, "recover" is associated with a different operation.

Reply:
> Longhorn Manager does not block the entire upgrade
This statement is incorrect. I've corrected it based on your suggestion.

Reply:
> Longhorn Manager does not block the entire upgrade - This statement is incorrect. 
Okay. I based that on the original draft: "This is because Longhorn manager will fail to block the entire upgrade."

Reply:
I see, the original draft was unclear and confusing. Thank you for pointing that out. PTAL and let me know if there are further suggestions.